test/e2e/tooltip.test.js

```diff
@@ -9,33 +9,61 @@ describe('vl-tooltip', async () => {
         return vlTooltipPage.load();
     });
 
-    it('als ik over een knop hover, zie ik de tooltip', async () => {
-        const button = await vlTooltipPage.getTopTooltipButton();
+    it('als gebruiker kan ik over een knop hoveren om de tooltip te zien', async () => {
```

Comment:
ik zou de testen opsplitsen. 
het zijn afzonderlijke features om een tooltip links, rechts, boven of onder te kunnen laten zien ...